sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_operation_type.hpp

```diff
@@ -19,30 +19,30 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
    * @brief An operation that can be performed with the key.
    *
    */
-  class KeyOperation final {
+  class KeyOperationType final {
```

Comment:
@gearama This type rename needs a change log entry since it is a breaking change.